lib/common/util.js

```diff
@@ -92,9 +92,14 @@ module.exports.extendGlobalConfig = extendGlobalConfig;
  * // [ 'Hi' ]
  */
 function arrayize(input) {
+  if (input == null) {
```

Comment:
Could we just rewrite as `input === null || input === undefined` for readability and to limit type coercion?